thoth/lab/utils.py

```diff
@@ -97,3 +97,21 @@ def on_import_error(package_name):
         importable.append(import_successful)
 
     return pd.DataFrame(data={'package': packages, 'version': versions, 'importable': importable})
+
+
+# Pandas utilities
+
+def str_join(df: pd.DataFrame, cols: list, sep: str = ''):
```

Comment:
Please don't use mutable default arguments, do something like:

```python
def str_join(df: pd.DataFrame, cols: list, sep: str = None):
    sep = sep or ''
```

Reply:
```python
# Pandas utilities
```
Also if you plan to have more pandas utilities, feel free to create a standalone file for that. I leave structure up to you.

Reply:
Uhm, I am aware of this topic, but it's not an issue in this case, is it? It's a string literal, it's not assignable, so it does not change its value with successive function calls.